### PR TITLE
[mds-db] [mds-policy-author] Feature/jane/create geos summaries endpoint

### PR DESCRIPTION
**Pull request by janedotx**
## PR Checklist

 - [ ] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
 - [ ] briefly describe the changes in this PR
 - [ ] mark as draft if should not be merged
 - [ ] write tests for all new functionality

## Impacts
- [ ] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance
- [ ] Daily
- [ ] Native
- [ ] Policy Author

### CODE REVIEW

File: packages/mds-db/index.ts [hunk 1]
```diff
@@ -1119,14 +1119,20 @@ async function readSingleGeography(geography_id: UUID): Promise<Geography> {
...
+async function readGeographies(params?: { get_read_only?: boolean; summary?: boolean }): Promise<Geography[]> {
   // use params to filter
   // query on ids
   // return geographies
   try {
     const client = await getReadOnlyClient()
 
-    let sql = `select * from ${schema.TABLE.geographies}`
+    const cols =
+      params && params.summary
```
Comment (avatarneil) [inline packages/mds-db/index.ts]: I am *SO* looking forward to TS 3.7 `params?.summary`

File: packages/mds-db/index.ts [hunk 2]
```diff
@@ -1119,14 +1119,20 @@ async function readSingleGeography(geography_id: UUID): Promise<Geography> {
   }
 }
 
-async function readGeographies(params?: { get_read_only?: boolean }): Promise<Geography[]> {
+async function readGeographies(params?: { get_read_only?: boolean; summary?: boolean }): Promise<Geography[]> {
```
Comment (drtyh2o) [inline packages/mds-db/index.ts]: Why snake  case? Does `get_read_only` originate from a query string?

File: packages/mds-db/index.ts (hunk 1 above)
Comment (drtyh2o) [reply packages/mds-db/index.ts]: See above... `Partial` with initializer and defaults avoids this.

File: packages/mds-policy-author/api.ts [hunk 3]
```diff
@@ -449,6 +449,17 @@ function api(app: express.Express): express.Express {
     }
   )
 
+  app.get(pathsFor('/geographies'), checkAccess(scopes => scopes.includes('policies:read')), async (req, res) => {
+    const summary = req.query.summary === 'true'
+    try {
+      const geographies = await db.readGeographies({ summary: Boolean(summary) })
```
Comment (drtyh2o) [inline packages/mds-policy-author/api.ts]: `summary` is already a boolean ... the `Boolean(summary)` is unnecessary.

**Pull request merged**